standalone-metastore/metastore-server/src/main/resources/package.jdo

```diff
@@ -433,7 +434,7 @@
       <field name="numBuckets">
         <column name="NUM_BUCKETS" jdbc-type="integer"/>
       </field>
-      <field name="serDeInfo" dependent="true">
```

Comment:
why is this field removed?

Reply:
Only the "dependent" property is removed. This patch makes serDeInfo be shared between StorageDescriptors, so it is not "dependent" any more.